config/controller/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/elasticache-controller
-  newTag: 0.1.0
+  newTag: 0.0.31
```

Comment:
Please make sure to `git pull --tags` and regenerate the contorller

Reply:
I'm not done making the full changes yet. I was just testing the pipeline.

Reply:
Actually, this is caused by https://github.com/aws-controllers-k8s/elasticache-controller/pull/140. `git describe --tags` now results in `0.1.0`.